plugins/armorblox/plugin.spec.yaml

```diff
@@ -0,0 +1,203 @@
+  final_detection_tag:
+    detection_tag_id:      
+      title: Detection tag ID
+      description: Detection tag ID
+      type: string
+      required: false
+    detection_tag_name:
+      title: Detection tag name
+      description: Detection tag name
+      type: string
+      required: false    
+  engagement:
+    fwd_mail_count:
+      title: Mail Count
+      description: Mail Count
+      type: string
+      required: false
+    reply_mail_count:
+      title: Mail Count
+      description: Mail Count
+      type: string
+      required: false
+  incident:
+    priority:
+      title: Priority
+      description: Priority of the incident
+      type: string
+      required: false
+    tagged:
+      title: Is email tagged
+      description: Is email tagged
+      type: boolean
+      required: false
+    date:
+      title: Incident Date
+      description: 
+      type: date
+      required: false
+    policy_names:
+      title: policy_names
+      description: List of policies
+      type: '[]string'
+      required: false
+    title:
+      title: Subject
+      description: Subject
+      type: 'string'
+      required: false          
+    resolution_state:
+      title: resolution_state
+      description: Resolution State
+      type: string
+      required: false
+    object_type:
+      title: Object Type
+      description: Object Type
+      type: string
+      required: false 
+    id:
+      title: Incident ID
+      description: Incident ID
+      type: string
+      required: false
+    app_name:
+      title: App Name
+      description: App Name
+      type: string
+      required: false 
+    external_senders:
+      title: External senders
+      description: List of external senders
+      type: "[]string"
+      required: 
+    folder_categories:
+      title: Folder categories
+      description: Folder categories
+      type: "[]string"
+      required: false
+    scl_score:
+      title: SCL Score
+      description: 
+      type: integer
+      required: false
+    incident_type:
+      title: Incident Type
+      description: Incident Type
+      type: string
+      required: 
+    engagements:
+      title: Engagements
+      description: Engagements 
+      type: engagement
+      required: false
+    remediation_actions:
+      title: Remediation Action
+      description: Remediation Action
+      type: "[]string"
+      required: false
+    users:
+      title: users
+      description: List of users
+      type: "[]user"
+      required: false
+    external_users:
+      title: External users
+      description: List of external users
+      type: "[]user"
+      required: false
+    final_detection_tags:
+      title: Detection tags
+      description: Detection tags 
+      type: "[]final_detection_tag"
+      required: false
+
+connection:
+  tenant_name:
+    title: Tenant Name
+    description: Armorblox Tenant Name
+    type: string
+    required: true
+    example: outbound-integrations
```

Comment:
Please clean up references to outbound-integrations everywhere in this PR

Reply:
Okay.